utilities/transactions/write_prepared_txn.cc

```diff
@@ -136,7 +144,8 @@ Status WritePreparedTxn::CommitInternal() {
     // Commit the data that is accompnaied with the commit marker
     // TODO(myabandeh): skip AddPrepared
     wpt_db_->AddPrepared(commit_seq);
-    wpt_db_->AddCommitted(commit_seq, commit_seq);
+    uint64_t commit_seq_2 = GetACommitSeqNumber(commit_seq);
```

Comment:
Same question, does this increment of last_to_be_written_seq corresponds to a commit marker in WAL?

Reply:
No. It does not have any entry in the WAL. I am actually currently reviewing the code in TransactionLogIteratorImpl to make sure that this gap in the WAL sequence numbers do not break any logic.